modules/ting_new_materials/plugins/content_types/new_materials.inc

```diff
@@ -28,79 +28,122 @@ function ting_new_materials_content_type_render($subtype, $conf, $panel_args, $c
     // Get the node.
     $node = $context[0]->data;
 
-    // As we are using a pager, we need to find the current page.
```

Comment:
Lots of changes to this module. How is this related to the theme? Shouldn't it be in a separate commit/issue?